installer.sh

```diff
@@ -0,0 +1,2315 @@
+  if command -v ipcalc &>/dev/null; then
+    network=$(ipcalc "$base_ip/$prefix" | grep -w 'Network' | awk '{print $2}')
+  else
+    # Fallback: simple /24 assumption
+    network="${base_ip%.*}.0/$prefix"
+  fi
+  echo "$network"
+}
+
+
+
+configure_nfs_server() {
+    local tracker_key="nfs_configured"
+    local title="NFS Storage Configuration"
+    if is_configured "$tracker_key"; then
+        log "NFS storage is already configured. Skipping setup."
+        return 0
+    fi
+    log "Starting NFS storage configuration..."
+
+    if [[ -d "/export" ]] && grep -q "^/export " /etc/exports; then
+        show_dialog "info" "$title" "NFS is already configured. Skipping setup."
+        set_tracker_field "$tracker_key" "yes"
+        return 0
+    fi
+
+    local export_cidr
+    export_cidr=$(get_export_cidr)
+    # Step 1: Create exports and directories
+    mkdir -p /export/primary /export/secondary
+    if ! grep -q "^/export " /etc/exports; then
+        echo "/export  ${export_cidr}(rw,async,no_root_squash,no_subtree_check)" >> /etc/exports
+    fi
+
+    exportfs -a
+
+    # Step 2: Configure ports and services based on distro
+    if [[ "$OS_TYPE" =~ ^(ubuntu|debian)$ ]]; then
+        sed -i -e 's/^RPCMOUNTDOPTS="--manage-gids"$/RPCMOUNTDOPTS="-p 892 --manage-gids"/g' /etc/default/nfs-kernel-server
+        sed -i -e 's/^STATDOPTS=$/STATDOPTS="--port 662 --outgoing-port 2020"/g' /etc/default/nfs-common
+        grep -q 'NEED_STATD=yes' /etc/default/nfs-common || echo "NEED_STATD=yes" >> /etc/default/nfs-common
+        sed -i -e 's/^RPCRQUOTADOPTS=$/RPCRQUOTADOPTS="-p 875"/g' /etc/default/quota
+
+        systemctl restart nfs-kernel-server
+        SERVICE_STATUS=$?
+    
+    elif [[ "$OS_TYPE" =~ ^(rhel|centos|ol|rocky|almalinux)$ ]]; then
+    # Set ports in /etc/sysconfig/nfs if needed
+    cat <<EOF >> /etc/sysconfig/nfs
+MOUNTD_PORT=892
+STATD_PORT=662
+STATD_OUTGOING_PORT=2020
+LOCKD_TCPPORT=32803
+LOCKD_UDPPORT=32769
+RQUOTAD_PORT=875
+EOF
+
+    systemctl enable --now rpcbind nfs-server
+    systemctl restart nfs-server
+    SERVICE_STATUS=$?
+
+    # Open firewall ports if firewalld is running
+    if systemctl is-active --quiet firewalld; then
+      firewall-cmd --permanent --add-service=nfs
+      firewall-cmd --permanent --add-service=mountd
+      firewall-cmd --permanent --add-service=rpc-bind
+      firewall-cmd --reload
+    fi
+  else
+    show_dialog "info" "$title"  "Unsupported distribution: $OS_TYPE"
+    return 1
+  fi
+
+  # Step 3: Final result
+  if [[ $SERVICE_STATUS -eq 0 ]]; then
+    exports_list=$(exportfs)
+    show_dialog "info" "$title" "NFS Server configured and restarted successfully.\n\nCurrent exports:\n$exports_list"
+    set_tracker_field "$tracker_key" "yes"
+    log "NFS server configured successfully."
+  else
+    show_dialog "info" "$title" "Failed to restart NFS server. Please check the service logs."
+  fi
+}
+
+setup_management_server_database() {
```

Comment:
```suggestion
configure_management_server_database() {
```

in line with other related methods